meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -146,12 +146,12 @@ python prepare_excluded_directories() {
             for entry in os.listdir(full_path):
                 full_entry = os.path.join(full_path, entry)
                 if os.path.isdir(full_entry) and not os.path.islink(full_entry):
-                    shutil.rmtree(full_entry)
+                    subprocess.check_call(["rm", "-rf", full_entry])
                 else:
-                    os.remove(full_entry)
+                    subprocess.check_call(["rm", "-f", full_entry])
         else:
             # Delete whole directory.
-            shutil.rmtree(full_path)
+            subprocess.check_call(["rm", "-rf", full_entry])
```

Comment:
```suggestion
            subprocess.check_call(["rm", "-rf", full_path])
```

Reply:
Wow! Nicely spotted! I suppose this is never called for us, because all our entries end in a slash.